src/shared/output-mode.c

```diff
@@ -1,8 +1,32 @@
 /* SPDX-License-Identifier: LGPL-2.1-or-later */
 
+#include <stdlib.h>
+
+#include "log.h"
 #include "output-mode.h"
 #include "string-table.h"
 
+void getenv_output_mode(OutputMode *ret) {
```

Comment:
please do not call this "ret", since it's not really a return value. i.e. we'll return without failure here and not touch the thing. just call it `output_mode` or so